module_utils/network_lsr/nm_provider.py

```diff
@@ -1,6 +1,10 @@
 # SPDX-License-Identifier: BSD-3-Clause
 """ Support for NetworkManager aka the NM provider """
 
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
```

Comment:
Is 'type' the default 'metaclass' in python ?

Reply:
> Is 'type' the default 'metaclass' in python ?

https://docs.ansible.com/ansible/2.9/dev_guide/testing/sanity/metaclass-boilerplate.html